exchanges/bybit/bybit_test.go

```diff
@@ -3060,6 +3036,172 @@ func TestPushData(t *testing.T) {
 	}
 }
 
+func TestWSHandleAuthenticatedData(t *testing.T) {
+	t.Parallel()
+
+	b := new(Bybit) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
+	require.NoError(t, testexch.Setup(b), "Test instance Setup must not error")
+	b.API.AuthenticatedSupport = true
+	b.API.AuthenticatedWebsocketSupport = true
+	b.SetCredentials("test", "test", "", "", "", "")
+	testexch.FixtureToDataHandler(t, "testdata/wsAuth.json", func(ctx context.Context, r []byte) error {
+		if bytes.Contains(r, []byte("%s")) {
+			r = fmt.Appendf(nil, string(r), optionsTradablePair.String())
+		}
+		return b.wsHandleAuthenticatedData(ctx, r)
+	})
+	close(b.Websocket.DataHandler)
+	require.Len(t, b.Websocket.DataHandler, 6, "Should see correct number of private messages")
+
+	i := 0
+	for data := range b.Websocket.DataHandler {
+		i++
+		switch v := data.(type) {
+		case WsPositions:
+			require.Len(t, v, 1, "must see 1 position")
+			assert.Zero(t, v[0].PositionIdx, "PositionIdx should be 0")
+			assert.Zero(t, v[0].TradeMode, "TradeMode should be 0")
+			assert.Equal(t, int64(41), v[0].RiskID, "RiskID should match")
```

Comment:
🦍  The wording I've used so far is "X should be correct".
"match" feels overloaded, because of times we actually mean that some function or method called match returned true, etc.
Don't have to action, I'm just being pedantic/consistent.